components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -159,6 +162,26 @@ import { Debounce } from 'vue-debounce-decorator'
 import AvailableActions from './AvailableActions.vue'
 
 @Component<GalleryItem>({
```

Comment:
Not sure if it works since this is component cc @roiLeo

Reply:
Yup this still work at component level.

ref vue-meta [doc ](https://vue-meta.nuxtjs.org/faq/#how-is-metainfo-resolved)
& nuxt [doc ](https://nuxtjs.org/docs/features/meta-tags-seo)
> To avoid any duplication when used in child components, please give a unique identifier with the hid key to the meta description. This way vue-meta will know that it has to overwrite the default tag.

Reply:
Can we resolve this?